elevate-cpanel

```diff
@@ -2820,7 +2820,9 @@ sub _blocker_ea4_profile ($self) {
 
     INFO("Checking EasyApache profile compatibility with AlmaLinux 8.");
 
-    return unless backup_ea4_profile();
+    my $backup = backup_ea4_profile();
+    return         unless $backup;
+    unlink $backup unless $self->{_abort_on_first_blocker};
```

Comment:
this is correct but this logic should belong to `backup_ea4_profile` itself
and we should convert as a method and call it `$self->backup_ea4_profile`

Reply:
`backup_ea4_profile` is now a method.